core/src/main/scala/caliban/schema/ArgBuilder.scala

```diff
@@ -85,49 +89,50 @@ object ArgBuilder extends ArgBuilderInstances {
 }
 
 trait ArgBuilderInstances extends ArgBuilderDerivation {
+  private def error(msg: String): ExecutionError = ExecutionError(msg, innerThrowable = Some(InvalidInputArgument))
```

Comment:
If we're always writing `Can't build an A from B`, how making A and B parameters and define the message once?